src/System.Net.Http/tests/FunctionalTests/HttpClientMiniStressTest.cs

```diff
@@ -16,9 +16,8 @@ namespace System.Net.Http.Functional.Tests
 
     public class HttpClientMiniStress : HttpClientTestBase
     {
-        private static bool HttpStressEnabled => Configuration.Http.StressEnabled;
-
-        [ConditionalTheory(nameof(HttpStressEnabled))]
+        // IsStressModeEnabled is true when the environment variable COREFX_STRESS is set to 1 or true
```

Comment:
I don't think these comments are necessary, but otherwise looks good.  You might see if adding this comment instead as an XML comment on IsStressModeEnabled (and ensuring that project builds an XML comment file) produces a better experience.